lib/ex_machina.ex

```diff
@@ -4,6 +4,8 @@ defmodule ExMachina do
 
   ## Examples
 
+  Put this somewhere like `test/support/ex_machina.ex`
```

Comment:
Maybe `Create a module for your factories. Often somewhere like `test/support/machina.ex`?